specification/cognitiveservices/data-plane/Face/stable/v1.0/Face.json

```diff
@@ -967,7 +967,7 @@
     },
     "/persongroups/{personGroupId}/persons/{personId}/persistedfaces": {
```

Comment:
One last comment, we should also unify the description for adding faces by `?overload=stream`.

Reply:
Please remove all “LARGE” in the description.